lib/rubocop/server/cli.rb

```diff
@@ -22,7 +22,13 @@ class CLI
       STATUS_ERROR = 2
 
       SERVER_OPTIONS = %w[
-        --server --no-server --server-status --restart-server --start-server --stop-server
+        --server
+        --no-server
+        --server-status
+        --restart-server
+        --start-server
+        --stop-server
+        --spawn-server
```

Comment:
Why do we need a separate command for this? Can't we just have a switch inside `--server/start-server` for this?